docs/coordinator/api/remote.md

```diff
@@ -0,0 +1,68 @@
+# API
+
+**Please note:** This documentation is a work in progress and more detail is required.
+
+## Remote Write
+
+Write a Prometheus Remote write query to M3.
+
+### URL
+
+`/api/v1/prom/remote/write`
+
+### Method
+
+`POST`
+
+### URL Params
+
+None.
+
+### Header Params
+
+#### Optional
+
+--8<--
+docs/common/headers_optional_read_write.md
```

Comment:
... I wish I'd known this was a feature

Reply:
I had to turn it on by adding `pymdownx.snippets` to the markdown extensions, but yeah we should make use of this now.